engine/api/src/main/java/org/apache/cloudstack/engine/subsystem/api/storage/StoragePoolAllocator.java

```diff
@@ -45,10 +51,12 @@
      * @param ExcludeList
      *            avoid
      * @param int returnUpTo (use -1 to return all possible pools)
+     * @param boolean bypassStorageTypeCheck
```

Comment:
why add javadoc without description? please add a short sentence on how to use it, or remove it.

Reply:
@DaanHoogland done